apps/emqx/src/emqx_broker.erl

```diff
@@ -555,8 +567,10 @@ do_dispatch(Topic, #delivery{message = Msg}) ->
         0 ->
             ok = emqx_hooks:run('message.dropped', [Msg, #{node => node()}, no_subscribers]),
             ok = inc_dropped_cnt(Msg),
+            emqx_otel_trace:event(message_dropped),
```

Comment:
Would it make sense to integrate otel events into emqx metrics? May be specifying the subset of metrics denoted for otel export as events.

Reply:
Thanks for the suggestion.
I added this event more like an example.
Some events may be not related to any of the emqx metrics, some may be not worth measuring.
I assume we can consider integrating events to metrics once we have more or less stable list of events we want to report..